app/models/manageiq/providers/vmware/infra_manager/inventory/saver.rb

```diff
@@ -36,6 +36,8 @@ def stop_thread(wait: true)
   # intend to queue up save_inventory from multiple calling threads a mutex
   # must be added around ensure_saver_thread
   def queue_save_inventory(persister)
+    _log.debug { "queueing save_inventory [#{persister.uuid}]" }
```

Comment:
NOTE that this will be logged after we see the collector logs about object updates but it is single threaded so it is easy enough to walk back in the logs to where it logs `Processing 1 updates...`